Makefile

```diff
@@ -1,8 +1,25 @@
-BIN := drone-google-cloudstorage
-IMG ?= plugins/$(BIN)
+.PHONY: clean deps test build docker
 
-docker: $(BIN) Dockerfile
-	docker build --rm -t $(IMG) .
+export GOOS ?= linux
+export GOARCH ?= amd64
+export CGO_ENABLED ?= 0
 
-$(BIN): $(wildcard *.go)
-	GOOS=linux GOARCH=amd64 go build
+CI_BUILD_NUMBER ?= 0
+
+LDFLAGS += -X "main.buildDate=$(shell date -u '+%Y-%m-%d %H:%M:%S %Z')"
+LDFLAGS += -X "main.build=$(CI_BUILD_NUMBER)"
```

Comment:
> Exactly my point. How are you going to get that info at the moment? It would make sense to add related/dependent changes in one PR as opposed to scattered across unrelated changes.

Sure, this is something @gtaylor can still add to the first line of `main` function.